sql/core/src/test/scala/org/apache/spark/sql/execution/SQLViewSuite.scala

```diff
@@ -903,4 +903,16 @@ abstract class SQLViewSuite extends QueryTest with SQLTestUtils {
       }
     }
   }
+
+  test("SPARK-37569: view should report correct nullability information for nested fields") {
```

Comment:
can we move the test to `SQLViewTestSuite`? so that we can test it with temp, global temp and permanent views.

Reply:
Done